src/commands/cloud-run/constants.ts

```diff
@@ -0,0 +1,22 @@
+import {SITE_ENV_VAR} from '../../constants'
+
+const NODE_OPTIONS_ENV_VAR = 'NODE_OPTIONS'
+const DD_SERVICE_ENV_VAR = 'DD_SERVICE'
+const DD_PROJECT_ENV_VAR = 'DD_ENV'
```

Comment:
I thinik these should be abstracted from every project into general constants: https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging

Reply:
Agreed. Is this what you were thinking?

<img width="675" alt="Screenshot 2023-07-27 at 11 27 04 AM" src="https://github.com/DataDog/datadog-ci/assets/29668820/15366982-8726-4400-b4b4-3680ec29dcae">